README.md

```diff
@@ -41,12 +41,15 @@ conan remote add ess-dmsc https://artifactory.esss.lu.se/artifactory/api/conan/e
 
 The ESS remote is required for QtColorWidgets. In case of changes to their deployment infrastructure, check for updates [here](https://github.com/ess-dmsc/conan-configuration).
 
+Changes in conan v2 require the cloning of the [cmake-conan](https://github.com/conan-io/cmake-conan/tree/develop2) repository, and use of the develop2 branch.
+Once the repository is cloned and the develop2 branch checked out, this repo can be configured as shown.
```

Comment:
I think it would be great to specify a oneliner command doing the git clone of already the specific branch.

Reply:
How about adding the cmake-conan repo as a submodule? We can then set the cmake option in the CMakeLists file as we know the full path to the necessary file.

Reply:
Sounds good! We could also try FetchContent instead of a submodule? https://cmake.org/pipermail/cmake/2019-June/069709.html

Reply:
I would to not have either submodule or assumption of anything cloned or installed to the system that does not come through a common package manager such as apt/pip/brew. From my perspective `cmake-conan` loses its utility for conan2, the way they have made it.
How about just having the user run the conan command manually prior to building? That's one extra line, but that's still less than submodule shenanigans.

Reply:
Thanks php1ic.  I tried your branch, however I cannot compile on Ubuntu 22, it says:

```
======== Finalizing install (deploy, generators) ========
conanfile.txt: Writing generators to /tmp/nuclei/build/build/conan
conanfile.txt: Generator 'CMakeDeps' calling 'generate()'
conanfile.txt: Generator 'CMakeToolchain' calling 'generate()'
conanfile.txt: CMakeToolchain generated: conan_toolchain.cmake
conanfile.txt: Preset 'conan-release' added to CMakePresets.json. Invoke it manually using 'cmake --preset conan-release' if using CMake>=3.23
conanfile.txt: If your CMake version is not compatible with CMakePresets (<3.23) call cmake like: 'cmake <path> -G "Unix Makefiles" -DCMAKE_TOOLCHAIN_FILE=/tmp/nuclei/build/build/conan/conan_toolchain.cmake -DCMAKE_POLICY_DEFAULT_CMP0091=NEW -DCMAKE_BUILD_TYPE=Release'
conanfile.txt: CMakeToolchain generated: CMakePresets.json
conanfile.txt: CMakeToolchain generated: ../../../CMakeUserPresets.json
conanfile.txt: Generating aggregated env files
conanfile.txt: Generated aggregated env files: ['conanbuild.sh', 'conanrun.sh']
Install finished successfully
-- CMake-Conan: CONAN_GENERATORS_FOLDER=/tmp/nuclei/build/build/conan
-- CMake-Conan: CONANFILE=/tmp/nuclei/conanfile.txt
CMake Error at /tmp/cmake-conan/conan_provider.cmake:278 (find_package):
  Could not find a package configuration file provided by "Threads" with any
  of the following names:

    ThreadsConfig.cmake
    threads-config.cmake

  Add the installation prefix of "Threads" to CMAKE_PREFIX_PATH or set
  "Threads_DIR" to a directory containing one of the above files.  If
  "Threads" provides a separate development package or SDK, be sure it has
  been installed.
Call Stack (most recent call first):
  CMakeLists.txt:7 (find_package)

```

Reply:
> I would to not have either submodule or assumption of anything cloned or installed to the system that does not come through a common package manager such as apt/pip/brew. From my perspective `cmake-conan` loses its utility for conan2, the way they have made it. How about just having the user run the conan command manually prior to building? That's one extra line, but that's still less than submodule shenanigans.

Sounds good to me. What would be then the exact install command for conan?

Reply:
This configuration issue with the `Threads` package could be another reason to move away from `cmake-conan`. When I made the PR the tip of the `develop2` branch was at [38a78aaf](https://github.com/conan-io/cmake-conan/commit/38a78aaf3a5567bd29932bf23e9) but the tip now also gives me that error.

Running conan externally, then cmake, does not give this issue. I'll update the README with the conan command to run and modified cmake commands.

CMake introduced [presets](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html) to simply configuration and conan 2 makes use of that. It arrived in cmake 3.19, the newest release is 3.26 and the latest [ubuntu LTS](https://packages.ubuntu.com/search?keywords=cmake) is on 3.22. Any issues with me using presets in the instructions?

Reply:
Good idea the presets, thanks!

Reply:
I should have read the conan output more carefully. Presets arrived in 3.19, but conan uses functionality from 3.23 so they can't be used by everyone at the moment.

The cmake command looks worse than it is, but if we want to make it super easy, we could write a simple build script?

Reply:
Where is the dividing line for who has the new CMake? I don't see any need to actively support anything older than what the latest Ubuntu LTS has.

Reply:
Ubuntu LTS has 3.22 but conan uses 3.23, i think that was the issue

Reply:
Correct, 3.23 introduced functionality that conan makes use of and the Ubuntu LTS (and thus the CI image) is on 3.22